### PR TITLE
Article App GET and POST Request

### PR DESCRIPTION
**Pull request by Aliqaan**
### Requirements listed in [#103](https://github.com/bounswe/bounswe2022group6/issues/103) are met.
The app now provides support for GET and POST requests and a user interface. The implementation for the app needs to be reviewed by 2 people before merging as discussed in meeting 15.

### CODE REVIEW

**Review by haticerk — APPROVED** [review]
I reviewed your work everything looks great

**Comment by artun-akdogan** [comment]
Except for the points I mentioned, I reviewed your application and it looks great.

File: practice_app/apps/article/views.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
+from asyncio.windows_events import NULL
```
Comment (artun-akdogan) [inline practice_app/apps/article/views.py:1]: This line causes an error on Linux. As we will use docker to publish the application and NULL is not used in this file, please remove this line.

File: practice_app/apps/article/migrations/0001_initial.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.5 on 2022-05-13 13:10
```
Comment (artun-akdogan) [inline practice_app/apps/article/migrations/0001_initial.py:1]: The migrations folder is created by Django. This folder will be added to .gitignore on future commits. This folder is redundant, please remove it.

File: practice_app/apps/article/views.py (hunk 1 above)
Comment (Aliqaan) [reply practice_app/apps/article/views.py]: Looks like I forgot to delete it. Now it is removed.

File: practice_app/apps/article/migrations/0001_initial.py (hunk 2 above)
Comment (Aliqaan) [reply practice_app/apps/article/migrations/0001_initial.py]: This folder is also removed with the latest push.

**Review by artun-akdogan — APPROVED** [review]
The application looks great

**Pull request merged**